arwen/interface.go

```diff
@@ -17,13 +17,10 @@ type StateStack interface {
 
 // ArgumentsParser defines the functionality to parse transaction data into arguments and code for smart contracts
 type ArgumentsParser interface {
-	GetArguments() ([][]byte, error)
-	GetCode() ([]byte, error)
+	GetFunctionArguments() ([][]byte, error)
```

Comment:
Keep it simple, without argument in function name

Reply:
See above comment.